src/types.md

```diff
@@ -459,16 +459,18 @@ fn main() {
 In this example, the trait `Printable` occurs as a trait object in both the
 type signature of `print`, and the cast expression in `main`.
 
+### Trait Object Lifetimes
```

Comment:
"Trait Object Lifetime Bounds"?

Reply:
Avoided messing with the verbiage of this section at all (I don't understand what it's conveying yet myself, as I haven't put much thought into it), but that's a better title for the section, so changed it.